audio-analysis/lib/etl/sqlConnection.py

```diff
@@ -3,14 +3,34 @@
 from secrets import db_username, db_password
 
 #### SQL CONNECTION INFO ####
-# Currently working on localhost
 # MAKE SURE YOU HAVE A DATABASE CALLED 'climactic_test'
 
-engine = create_engine('mysql+pymysql://' + db_username + ':' + db_password + '@climactic-test.cmikkru8vljn.us-east-1.rds.amazonaws.com:3306/climactic_test')
+
+def getEngine(isTest):
+    if isTest:
+        try:
+            engine = create_engine('mysql+pymysql://root:root@localhost/climactic_test')
+            engine.connect()
+            print("\nYou are connected to ", engine, "\n")
+        except Exception:
```

Comment:
Should we fail on Exception? whats the difference between `'mysql+pymysql://root@localhost/climactic_test'` and `'mysql+pymysql://root:root@localhost/climactic_test'`?

Reply:
we need to get to the bottom of default user for MySQL db on local. Somehow for Wega it is password "NO" and for me password "YES"

Reply:
Ahh I see. I feel kinda behind in all the sql db setup. Maybe next meeting we can go over it?

Reply:
yes. It really is a two click thing))